.storybook/stories/TableFiltering/components/DashboardTable.tsx

```diff
@@ -107,39 +107,39 @@ export const DashboardTable = forwardRef<HTMLTableElement, DashboardTableProps>(
 									width,
 									isSortable: isFieldSortable,
 								}) => {
-									if (!isTableSortable || !isFieldSortable) {
+									if (isTableSortable && isFieldSortable) {
```

Comment:
Rather than saying `if these conditions ARE NOT met`, it's a lot easier to read if statements when they are declared as `do this when these conditions ARE met`

Reply:
Haven't changed any of the rendering logic, just moved some things around as we have changed the `if` logic.